src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionTest.cs

```diff
@@ -547,6 +547,85 @@ public void GetSchema_Connection_Closed()
             }
         }
 
+        [Theory]
+        [InlineData(@"AttachDbFileName=C:\test\attach.mdf", @"AttachDbFileName=C:\test\attach.mdf")]
+        [InlineData(@"AttachDbFileName=C:\test\attach.mdf;", @"AttachDbFileName=C:\test\attach.mdf;")]
+        public void ConnectionString_AttachDbFileName_Plain(string value, string expected)
+        {
+            SqlConnection cn = new SqlConnection();
+            cn.ConnectionString = value;
+            Assert.Equal(expected, cn.ConnectionString);
+        }
+
+        [Theory]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
Why does this work on Windows only? Does it have to do with AppDomain? 

What happens on Linux? Can you add a test to run this on linux and the test should check for exceptions.

Reply:
Only Windows SQL Server SKUs allows you to use attachdbfilename! (SQL Server Express and LocalDB) - it is the full path (Path.GetFullPath) requirement that makes Linux test fail.